Pipeline/SimpleBuildReportDeploy/SimpleBuildReportDeploy.groovy

```diff
@@ -0,0 +1,292 @@
+@groovy.transform.BaseScript com.ibm.dbb.groovy.ScriptLoader baseScript
```

Comment:
Please remove unused imports.

For example `org.apache.http.entity.FileEntity` would break the execution of the script with DBB 2.0.

Reply:
This is done and created a new pull request #148 .

Reply:
Please ignore #148 and consider #149 .